specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/UnivariateAnomalyDetector.json

```diff
@@ -0,0 +1,598 @@
+        },
+        "customInterval": {
+          "$ref": "#/definitions/CustomInterval",
+          "description": "Custom Interval is used to set non-standard time interval, for example, if the series is 5 minutes, request can be set as {\"granularity\":\"minutely\", \"customInterval\":5}."
+        },
+        "period": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Optional argument, periodic value of a time series. If the value is null or does not present, the API will determine the period automatically."
+        },
+        "stableTrendWindow": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Optional argument, advanced model parameter, a default stableTrendWindow will be used in detection."
+        },
+        "threshold": {
+          "type": "number",
+          "format": "float",
+          "description": "Optional argument, advanced model parameter, between 0.0-1.0, the lower the value is, the larger the trend error will be which means less change point will be accepted."
+        }
+      }
+    },
+    "ChangePointDetectResponse": {
+      "type": "object",
+      "description": "The response of change point detection.",
+      "properties": {
+        "period": {
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true,
+          "description": "Frequency extracted from the series, zero means no recurrent pattern has been found."
+        },
+        "isChangePoint": {
+          "type": "array",
+          "description": "isChangePoint contains change point properties for each input point. True means an anomaly either negative or positive has been detected. The index of the array is consistent with the input series.",
+          "items": {
+            "type": "boolean",
+            "readOnly": true
+          }
+        },
+        "confidenceScores": {
+          "type": "array",
+          "description": "the change point confidence of each point",
+          "items": {
+            "type": "number",
+            "format": "float",
+            "readOnly": true
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "ApiVersion": {
+      "name": "ApiVersion",
+      "description": "Anomaly Detector API version (for example, v1.1).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "v1.1"
+      ],
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    }
```

Comment:
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-versioning

✅ DO use a required query parameter named api-version on every operation for the client to specify the API version.

Reply:
Discussed with Mike and Johan before, we can keep this version in path to align with public preview.

Reply:
Just to be clear, we're viewing MVAD as an _extension_ of AD, an existing GA service with the version in the path. So we're keeping version in the path here to avoid breaking changes to a GA service.